pkgs/build-support/docker/default.nix

```diff
@@ -434,6 +434,8 @@ rec {
         if [[ -n "$fromImage" ]]; then
           echo "Unpacking base image..."
           tar -C image -xpf "$fromImage"
+          # Do not import the base image configuration
+          rm -f image/*.json
```

Comment:
I also did a patch to make `buildImage` produce image spec v1.2 compatible images, see https://github.com/holidaycheck/nixpkgs/commit/ece423325be64bbf0d0d9424d0441e485617faad

Before submitting this patch to upstream I still want to implement a test for `dockerTools`.